pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -43,6 +43,25 @@ pub mod pallet {
 	};
 	use sp_std::vec::Vec;
 
+	#[derive(Debug, Encode, Decode, MaxEncodedLen, TypeInfo)]
+	pub struct SeasonConfigs<Balance, BlockNumber> {
+		pub mint_available: bool,
+		pub mint_fees: Balance,
```

Comment:
Let's try getting https://github.com/ajuna-network/Ajuna/pull/74 merged in first so this becomes:
```suggestion
		pub mint_fees: MintFees<Balance>,
```

Reply:
Finished rebase and code adaptations!